### PR TITLE
[PIE-1819] Documentation for Analytics REST API Suites#update and Suite#destroy

### PR DESCRIPTION
**Pull request by amybiyuliu**
Add documentation about test analytics Suites#update and Suite#destroy API endpoints

Suites#update:
Screenshot 2023-06-26 at 4 35 20 pm
Suite#destroy:

Screenshot 2023-06-26 at 4 38 34 pm

### CODE REVIEW

File: pages/apis/rest_api/analytics/suites.md [hunk 1]
```diff
@@ -97,3 +97,50 @@ Optional [request body properties](/docs/api#request-body-properties):
...
+  "web_url": "https://buildkite.com/organizations/my_great_org/analytics/suites/jasmine",
+  "default_branch": "main"
+}
+```
+
+Optional [request body properties](/docs/api#request-body-properties):
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>name</code></th><td>Name of the new suite.<br><em>Example:</em> <code>"Jasmine"</code>.</td></tr>
```
Comment (swebb) [inline pages/apis/rest_api/analytics/suites.md:127]:
```suggestion
  <tr><th><code>name</code></th><td>Name of the suite.<br><em>Example:</em> <code>"Jasmine"</code>.</td></tr>
```

File: pages/apis/rest_api/analytics/suites.md [hunk 2]
```diff
@@ -97,3 +97,50 @@ Optional [request body properties](/docs/api#request-body-properties):
...
+
+Required scope: `write_suites`
+
+Success response: `200 OK`
+
+## Delete a suite
+
+```bash
+curl -X DELETE \
+  https://api.buildkite.com/v2/analytics/organizations/{org.slug}/suites{suite.slug} \
```
Comment (mbelton-buildkite) [inline pages/apis/rest_api/analytics/suites.md:141]: Style: I don't believe this is necessary.

```suggestion
  https://api.buildkite.com/v2/analytics/organizations/{org.slug}/suites{suite.slug}
```

**Review by mbelton-buildkite — APPROVED** [review]
One minor style tweak, otherwise, looks good to me with @swebb's suggestions. 😊

File: pages/apis/rest_api/analytics/suites.md [hunk 3]
```diff
@@ -97,3 +97,50 @@ Optional [request body properties](/docs/api#request-body-properties):
...
+```
+
+Required scope: `write_clusters`
```
[the remark this answers is not in the document]
Comment (amybiyuliu) [reply pages/apis/rest_api/analytics/suites.md]: whoops I copied it from pipelines docs and forgot to update the scope name. Good catch!

**Review by swebb — APPROVED** [review]

**Pull request merged**